### ISSUE TITLE
Detroit RailsBridge, September 11th-12th

### ISSUE THREAD

**Issue by crebma** [railsbridge/organizing#76]
We are just getting going on running the first Detroit RailsBridge workshop, and we are pretty excited about it! We are myself and @saragibby. We have nothing completed yet besides deciding when we want to do it. :)

**Comment by crebma** [comment]
We have found a venue (Grand Circus) and have started registering folks, and we have our instructors! Getting lots of good attention already. :)

**Comment by crebma** [comment]
Our workshop is almost completely full, but we still haven't found any sponsors, unfortunately. I tend to send a lot of request emails for sponsorships in general, but they're always to local companies - are there any companies that might be interested that aren't in southeast Michigan?

**Comment by rachelmyers** [comment]
Yes! I know there are others, but I can help along requests to GitHub. Here's the [form](https://community.github.com/?r=rachelmyers) to ask for :money_with_wings:, and I'd definitely mention that the workshop is quickly approaching and you're having trouble finding sponsors. :smile_cat:

**Comment by crebma** [comment]
Thank you so much, @rachelmyers!
